test-data/unit/check-overloading.test

```diff
@@ -5404,15 +5406,15 @@ def f_d(arg: str) -> None: ...
 @dec_d  # E: "int" not callable
 def f_d(arg): ...
 
-Bad1 = TypeVar('Good')  # type: ignore
+Bad1 = TypeVar(Bad1)  # type: ignore
 
 def dec_e(f: Bad1) -> Bad1: ...  # type: ignore
 
 @overload
 def f_e(arg: int) -> None: ...
 @overload
 def f_e(arg: str) -> None: ...
-@dec_e  # E: Bad1? not callable
+@dec_e
```

Comment:
whats this

Reply:
there was a test for a decorator with an error in it, i made typevars not fail completely when the name is wrong, so i changed it, i think it's still a valid test though